src/nspawn/nspawn.c

```diff
@@ -2666,13 +2666,12 @@ static int setup_journal(const char *directory) {
                 /* don't create parents here — if the host doesn't have
                  * permanent journal set up, don't force it here */
 
-                r = mkdir_errno_wrapper(p, 0755);
-                if (r < 0 && r != -EEXIST) {
+                if (mkdir(p, 0755) < 0 && errno != EEXIST) {
```

Comment:
```C
r = RET_NERRNO(mkdir(p, 0755));
if (r < 0 && r != -EEXIST) {
     …
```

Reply:
the code i added is shorter though?

Reply:
Yeah, but brevity is not everything. I think it's nicer to consistently use the wrapper everywhere and not use raw `errno` if possible.